specs/referrer-policy/index.html

```diff
@@ -961,14 +972,72 @@ <h3 class="heading settled" data-level="4.2" id="referrer-policy-delivery-meta">
     </dl>
 
   
-    <h3 class="heading settled" data-level="4.3" id="referrer-policy-delivery-implicit"><span class="secno">4.3. </span><span class="content">Implicit Delivery</span><a class="self-link" href="#referrer-policy-delivery-implicit"></a></h3>
+    <h3 class="heading settled" data-level="4.3" id="referrer-policy-delivery-referrer-attribute"><span class="secno">4.3. </span><span class="content">Delivery
+  via a <code>referrer</code> attribute</span><a class="self-link" href="#referrer-policy-delivery-referrer-attribute"></a></h3>
+
+
+    <p>A referrer policy may be set when a <code>referrer</code> attribute is
+  added to an <code><a data-link-type="element" href="https://html.spec.whatwg.org/#the-a-element">a</a></code>, <code><a data-link-type="element" href="https://html.spec.whatwg.org/#the-area-element">area</a></code>,
+  <code><a data-link-type="element" href="https://html.spec.whatwg.org/#the-img-element">img</a></code> or <code><a data-link-type="element" href="https://html.spec.whatwg.org/#the-iframe-element">iframe</a></code>
+  element, for example:</p>
+
+  
+    <pre class="example">&lt;a href="http://example.com" referrer="origin">
+</pre>
+
+
+    <p>The following values for the <code>referrer</code> attribute are valid,
+  and map to the listed <a data-link-type="dfn" href="#referrer-policy">referrer policy</a> values:</p>
```

Comment:
Is there any reason to limit to just these three? It might make sense to support all of the following here: no-referrer, no-referrer-when-downgrade, origin, origin-when-cross-origin, unsafe-url

I only need the three listed at the moment.